modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -1064,6 +1067,21 @@ public void postProcessModelProperty(CodegenModel model, CodegenProperty propert
         }
     }
 
+    @Override
+    public Mustache.Compiler processCompiler(final Mustache.Compiler compiler) {
+        return compiler
+            .withEscaper(Escapers.simple(new String[][] {
+                { "&", "&amp;" },
+                { "'", "&#39;" },
+                { "<", "&lt;" },
+                { ">", "&gt;" },
+                { "`", "&#x60;" }
+                // part of default Escaper - Escapers#HTML
+                //{ "\"", "&quot;" },
+                //{ "=",  "&#x3D;" }
+            }));
+    }
+
```

Comment:
To use the unescaped (original) value, what about using `{{{value}}}` instead of `{{value}}`  in the mustache template?